internal/proto/plugin/LICENSE

```diff
@@ -0,0 +1,365 @@
+Copyright (c) 2020 HashiCorp, Inc.
```

Comment:
This licenses the plugin protobuf files to MPL since their generated files are part of the sdk module.